azext_iot/central/_help.py

```diff
@@ -44,6 +57,257 @@ def load_central_help():
+            --destinations "[
+              {
+                'id': '{destinationid}',
+                'transform': '{ ApplicationId: .applicationId, Component: .component, DeviceName: .device.name }'
+              }
+            ]"
+    """
+
+    helps[
+        "iot central export update"
+    ] = """
+        type: command
+        short-summary: Update an export for an IoT Central application.
+        long-summary: Source is not allowed to update once export was created.
+        examples:
+        - name: Update an export from file
+          text: >
+            az iot central export update
+            --app-id {appid}
+            --export-id {exportid}
+            --content './filepath/payload.json'
+
+        - name: Update an export's display name and enable export from json payload
+          text: >
+            az iot central export update
+            --app-id {appid}
+            --export-id {exportid}
+            --content "{'displayName': 'Updated Export Name', 'enabled': true}"
+    """
+
+    helps[
+        "iot central export delete"
+    ] = """
+        type: command
+        short-summary: Delete an export for an IoT Central application.
+        examples:
+        - name: Delete an export
+          text: >
+            az iot central export delete
+            --app-id {appid}
+            --export-id {exportid}
+    """
+    _load_central_destination_help()
+
+
+def _load_central_destination_help():
+    helps[
+        "iot central export destination"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central export destinations.
+    """
+
+    helps[
+        "iot central export destination list"
+    ] = """
+        type: command
+        short-summary: Get the full list of export destinations for an IoT Central application.
+        examples:
+        - name: List all export destinations in an application
+          text: >
+            az iot central export destination list
+            --app-id {appid}
+    """
+
+    helps[
+        "iot central export destination show"
+    ] = """
+        type: command
+        short-summary: Get an export destination details
+        examples:
+        - name: Get an export destination details
+          text: >
+            az iot central export destination show
+            --app-id {appid}
+            --dest-id {destinationid}
+    """
+
+    helps[
+        "iot central export destination create"
+    ] = """
+        type: command
+        short-summary: Create an export destination for an IoT Central application.
+        examples:
+        - name: Create a webhook export destination with json payload
+          text: >
+            az iot central export destination create
+            --app-id {appid}
+            --dest-id {destinationid}
+            --name {displayname}
+            --url {url}
+            --type webhook@v1
+            --header "{\"x-custom-region\":{\"value\":\"westus\", \"secret\": false}}"
```

Comment:
Recommend consistent inline json example format -- bash syntax is ubiquitous and easiest to use.

Something to note is the `"{'property': 'value'}"` format (as used in other examples) is the CLI expanding flexibility/increasing ease of use for json input, but its technically not json, as json properties must be double-quoted  strings...this is more correct `'{"property": "value"}'` which would work in bash.

Reply:
Agreed, I will update these examples following bash format.